app/decorators/application_choice_export_decorator.rb

```diff
@@ -51,6 +51,10 @@ def rejection_reasons
     reasons&.map { |k, v| %(#{k.upcase}\n\n#{Array(v).join("\n\n")}) }&.join("\n\n")
   end
 
+  def domicile_country
+    DomicileResolver.country_for_hesa_code(application_form.domicile)
```

Comment:
domicile resolver is used in the `domicile` field in the application form so I think we can just use `application_form.domicile`

Reply:
This is a second stage mapping from HESA code to country name, we need first to get the code from `application_form.domicile` which calls a different method on `DomicileResolver` to get the corresponding HESA code. 
This is mapped slightly differently from ISO country code, we then get the relevant country name. 
Important we base this off `application_form.domicile` to get domestic UK countries and odd mappings like `Cyprus` and `Kosovo`.